pandas/__init__.py

```diff
@@ -49,7 +49,15 @@
 from pandas.tools.merge import (merge, ordered_merge,
                                 merge_ordered, merge_asof)
 from pandas.tools.pivot import pivot_table, crosstab
-from pandas.tools.plotting import scatter_matrix, plot_params
+
+# deprecate tools.plotting, plot_params and scatter_matrix on the top namespace
+import pandas.tools.plotting
```

Comment:
there are test for that in `pandas/tests/plotting/test_deprecated.py` (the top-evel deprecations of plot_params are in test_api.py as well)

added to https://github.com/pandas-dev/pandas/issues/6581